mne/io/eeglab/eeglab.py

```diff
@@ -119,6 +118,24 @@ def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
         Path or instance of montage containing electrode positions.
         If None, sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
+    eog : list | tuple | 'auto'
+        Names or indices of channels that should be designated
+        EOG channels. If 'auto', the channel names containing
+        ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict | None
+        The ids of the events to consider. If None (default),
+        an empty dict is used and `event_id_func` (see below) is called on
+        every event value. If dict, the keys will be mapped to trigger values
+        on the stimulus channel and only keys not in `event_id` will be handled
+        by `event_id_func`.
+        Keys are case-sensitive.
+        Example: {'SyncStatus': 1; 'Pulse Artifact': 3}.
```

Comment:
Better for rendering to do:
```
Example::

    {'SyncStatus': 1, 'Pulse Artifact': 3}

```
Or at least enclose it in double-backticks if you don't want to make it indented.